packages/calypso-e2e/src/element-helper.ts

```diff
@@ -96,7 +96,7 @@ export async function reloadAndRetry(
 ): Promise< void > {
 	for ( let retries = 3; retries > 0; retries -= 1 ) {
 		try {
-			await func( page );
+			return await func( page );
```

Comment:
This is to actually exit the retry loop once a successful run of the `func` is recorded.